TrainingExtensions/tensorflow/src/python/aimet_tensorflow/quantizer_info.py

```diff
@@ -75,12 +76,23 @@ class type to save pickle-able info pertaining to tensor quantizer
         """
 
         self.quant_op_name = quant_op_name
-        self.quant_scheme = tensor_quantizer_ref.getQuantScheme()
-        self.use_strict_symmetric = tensor_quantizer_ref.getStrictSymmetric()
-        self.use_unsigned_symmetric = tensor_quantizer_ref.getUnsignedSymmetric()
-        self.rounding_mode = tensor_quantizer_ref.roundingMode
-        self.is_encoding_valid = tensor_quantizer_ref.isEncodingValid
         self.quantizer_type = quantizer_type
+        self.num_channels = 0
+        if isinstance(tensor_quantizer_ref, list):
+            for tensor_quantizer in tensor_quantizer_ref:
+                self.quant_scheme = tensor_quantizer.getQuantScheme()
+                self.use_strict_symmetric = tensor_quantizer.getStrictSymmetric()
+                self.use_unsigned_symmetric = tensor_quantizer.getUnsignedSymmetric()
+                self.rounding_mode = tensor_quantizer.roundingMode
+                self.is_encoding_valid = tensor_quantizer.isEncodingValid
+                self.num_channels = len(tensor_quantizer_ref)
+                break
```

Comment:
I think I had a comment on this before. is tensor_quantizer_ref[0].getQuantScheme() more readable?

Reply:
Yes, I forgot to add it in this one place, will add as a part of the next PR